exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/send/thrift.py

```diff
@@ -66,11 +70,23 @@ def emit(self, batch: jaeger.Batch):
         self.client.emitBatch(batch)
         buff = self.buffer.getvalue()
         if len(buff) > self.max_packet_size:
-            logger.warning(
-                "Data exceeds the max UDP packet size; size %r, max %r",
-                len(buff),
-                self.max_packet_size,
-            )
+            if self.split_oversized_batches and len(batch.spans) > 1:
+                packets = math.ceil(len(buff) / self.max_packet_size)
+                div = math.ceil(len(batch.spans) / packets)
+                for packet in range(packets):
+                    start = packet * div
+                    end = (packet + 1) * div
+                    self.emit(
+                        jaeger.Batch(
+                            process=batch.process, spans=batch.spans[start:end]
```

Comment:
In some cases the last emit can produce a batch with empty spans? Can we avoid it? :slightly_smiling_face: 

For example, try with:
```
spans=4
buffer_length=131000
```

Reply:
Oh, thabk you! I've addressed this now.